Foundation/ByteCountFormatter.swift

```diff
@@ -354,14 +354,13 @@ open class ByteCountFormatter : Formatter {
     // A helper method to return the length of an int
     private func lengthOfInt(number: Int) -> Int {
         var num = abs(number)
-        var length: [Int] = []
+        var length = 0
         
```

Comment:
I see this code is only used to test for length of == 3 or > 3 but maybe length should be at least 1 for a value of 0?

Reply:
That is true, should I fix it in same PR or open a new one?

Reply:
I think this one is small enough that we can put that fix here as well.

Reply:
Done.